geomesa-accumulo/geomesa-accumulo-dist/src/main/assemblies/component.xml

```diff
@@ -40,10 +41,7 @@
             <useProjectArtifact>false</useProjectArtifact>
             <useTransitiveDependencies>false</useTransitiveDependencies>
             <includes>
-                <include>org.locationtech.geomesa:geomesa-accumulo-gs-plugin*</include>
                 <include>org.locationtech.geomesa:geomesa-process-wps*</include>
-                <include>org.locationtech.geomesa:geomesa-stream-gs-plugin*</include>
```

Comment:
Ok, slow morning questions...  

1.  Why are the gs-plugin out?
2.  Why is the geomesa-process still in?

Basically, what does this block do since we aren't creating a tar.gz?

Reply:
the tar.gz's are still being built, and they now get included as a fileset a few lines down, instead of as a dependency set here. had to make that change as they aren't 'attached' any more, so you can't retrieve them as a regular dependency. they aren't attached any more so that they don't get published to maven after being built.